core/templates/nukestudio.yml

```diff
@@ -22,9 +22,9 @@ keys:
 paths:
 
     # WORK AREAS
-    tk-nukestudio_project_work_area:            './{department}/user/@user_work'
-    tk-nukestudio_sequence_work_area:           './{department}/{Sequence}/user/@user_work'
-    tk-nukestudio_shot_work_area:               './{department}/{Sequence}/{Shot}/user/@user_work'
+    tk-nukestudio_project_work_area:            './user/@user_work/nukestudio'
```

Comment:
Should probably remove the `department` key above as well.

Reply:
Removing department key